web/app/templates/layout.html

```diff
@@ -103,9 +103,13 @@
   </nav>
   {% endwith %}
 
+  {% if settings.EMAIL_REQUIRES_VERIFICATION and user.is_authenticated and not user.has_verified_email %}
```

Comment:
I know this if statement sucks but I couldn't come up with a better way of putting this notification in every page, if you have a suggestion, I'd love to hear it.  Also putting it outside of the div was intentional

Reply:
This `if` is clean and self-explanatory. Not a problem at all.